client/src/components/Checkout/useStyles.ts

```diff
@@ -0,0 +1,24 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    minHeight: '100vh',
+    '& .MuiInput-underline:before': {
+      borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
+    },
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  authWrapper: {
```

Comment:
I don't think we are using this CSS class, can we please remove it?

Reply:
Sure.